ui-components/src/locales/general.json

```diff
@@ -617,8 +617,8 @@
   "listings.availableUnitsDescription": "Applicants will be reviewed in order until all vacancies are filled.",
   "listings.availableUnitsAndWaitlist": "Available units and waitlist",
   "listings.availableUnitsAndWaitlistDesc": "Once applicants fill all available units, additional applicants will be placed on the waitlist for <span class='t-italic'>%{number} units</span>",
-  "listings.availableUnit": "Available Unit",
```

Comment:
We can't change existing strings, because now our non-English keys don't match this new English string, but it looks like they are already translated. We should add a new string :)